### PR TITLE
Add information on the use of ThreadGuard Support class

### PR DESCRIPTION
**Pull request by bongosway**
To help highlight the correct use of the class to aid test design and architecture

### CODE REVIEW

File: docs_source_files/content/webdriver/support_classes.en.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
 ---
 title: "Support classes"
-weight: 5
+weight: 4
 ---
 
+Webdriver support classes are provided to make updating and maintaining your code easier. They provide a nice abstraction to make modelling HTML element(s) as domain objects easier,
```
[the remark this answers is not in the document]
Comment (diemol) [reply docs_source_files/content/webdriver/support_classes.en.md]: Can you please keep the line length to a maximum of 80 chars? This simplifies reviews.

File: docs_source_files/content/webdriver/support_classes.en.md [hunk 2]
```diff
@@ -1,5 +1,76 @@
...
+weight: 4
 ---
 
+Webdriver support classes are provided to make updating and maintaining your code easier. They provide a nice abstraction to make modelling HTML element(s) as domain objects easier, 
+also providing helper methods to make using such objects easy to reason about.
+We will learn about Locator Strategies, Events, PageFactory, LoadableComponent, ThreadGuard etc.
+
+Let's Start:
+
+## **ThreadGuard**
```
Comment (diemol) [inline docs_source_files/content/webdriver/support_classes.en.md]: This is only available in Java, we should mention that.

File: docs_source_files/content/webdriver/support_classes.en.md [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
+ThreadGuard checks that a driver is called only from the same thread that created it. 
+Threading issues especially when running tests in Parallel may have mysterious and hard to diagnose errors. 
+Using this wrapper prevents this category of errors and will raise an exception when it happens. 
+
+The following example simulate a clash of threads:
```
Comment (diemol) [inline docs_source_files/content/webdriver/support_classes.en.md]: Please only add Java code, no need to use the shortcode.

So, just use

```
    ```java 
    Here Java Code
    ```
```

File: docs_source_files/content/webdriver/support_classes.en.md [hunk 4]
```diff
@@ -1,5 +1,76 @@
 ---
 title: "Support classes"
-weight: 5
+weight: 4
 ---
 
+Webdriver support classes are provided to make updating and maintaining your code easier. They provide a nice abstraction to make modelling HTML element(s) as domain objects easier, 
+also providing helper methods to make using such objects easy to reason about.
+We will learn about Locator Strategies, Events, PageFactory, LoadableComponent, ThreadGuard etc.
```
[the remark this answers is not in the document]
Comment (bongosway) [reply docs_source_files/content/webdriver/support_classes.en.md]: Done

File: docs_source_files/content/webdriver/support_classes.en.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bongosway) [reply docs_source_files/content/webdriver/support_classes.en.md]: Sure. Thanks

File: docs_source_files/content/webdriver/support_classes.en.md (hunk 3 above)
Comment (bongosway) [reply docs_source_files/content/webdriver/support_classes.en.md]: Done

File: docs_source_files/content/webdriver/support_classes.en.md (hunk 2 above)
Comment (bongosway) [reply docs_source_files/content/webdriver/support_classes.en.md]: Info panel added

**Pull request merged**